api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/TPT_InitiationQueries.java

```diff
@@ -1237,7 +1237,7 @@ public static String getTPTInitiationTOTALS(
     valuesMap.put("1", hivCareProgram);
 
     String query =
-        "  SELECT (SELECT name FROM location WHERE location_id = :location) AS location, COUNT(TPT.person_id) AS total, CONCAT(:startDate,' Até ',:endDate) AS period "
+        "  SELECT (SELECT name FROM location WHERE location_id = :location) AS location, COUNT(TPT.person_id) AS total, CONCAT(DATE_FORMAT(:startDate, '%Y %M %d'),' Até ',DATE_FORMAT(:endDate, '%Y %M %d')) AS period "
```

Comment:
@cfaife  I see a difference in date format that Pinkie Had requested. (**dd-mm-yyyy** )